src/main/java/greencity/entities/Category.java

```diff
@@ -0,0 +1,36 @@
+package greencity.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
```

Comment:
Comment regarding all the Entities:
Are ALL the fields in ALL Entities not changable? 
If any value can change - it must be excluded from hashcode and equals method. 
Please consider something like that: @EqualsAndHashCode(exclude = {"changableField1", "changableField2", ...})